webapp/components/utils/InternalPages.js

```diff
@@ -0,0 +1,180 @@
+      htmlIdent: 'html.termsAndConditions',
+    },
+  }),
+  CODE_OF_CONDUCT: new PageParams({
+    name: 'code-of-conduct',
+
+    externalLink: null, // if string is defined and not empty it's dominating
+
+    internalPage: {
+      pageRoute: '/code-of-conduct', // static, don't change! internal page in case no external is defined
+      footerIdent: 'site.code-of-conduct', // localized string identifier
+      headTitleIdent: 'site.code-of-conduct', // localized string identifier
+      headlineIdent: 'site.code-of-conduct', // localized string identifier. on null it's hidden, on empty string default is used
+      hasContainer: true,
+      hasBaseCard: true,
+      hasLoginInHeader: true,
+      // in case internal page content is here 'webapp/locales/html/'
+      htmlIdent: 'html.codeOfConduct',
+    },
+  }),
+  DATA_PRIVACY: new PageParams({
+    name: 'data-privacy',
+
+    externalLink: null, // if string is defined and not empty it's dominating
+
+    internalPage: {
+      pageRoute: '/data-privacy', // static, don't change! internal page in case no external is defined
+      footerIdent: 'site.data-privacy', // localized string identifier
+      headTitleIdent: 'site.data-privacy', // localized string identifier
+      headlineIdent: 'site.data-privacy', // localized string identifier. on null it's hidden, on empty string default is used
+      hasContainer: true,
+      hasBaseCard: true,
+      hasLoginInHeader: true,
+      // in case internal page content is here 'webapp/locales/html/'
+      htmlIdent: 'html.dataPrivacy',
+    },
+  }),
+  FAQ: new PageParams({
+    name: 'faq',
+
+    externalLink: null, // if string is defined and not empty it's dominating
+
+    internalPage: {
+      pageRoute: '/faq', // static, don't change! internal page in case no external is defined
+      footerIdent: 'site.faq', // localized string identifier
+      headTitleIdent: 'site.faq', // localized string identifier
+      headlineIdent: 'site.faq', // localized string identifier. on null it's hidden, on empty string default is used
+      hasContainer: true,
+      hasBaseCard: true,
+      hasLoginInHeader: true,
+      // in case internal page content is here 'webapp/locales/html/'
+      htmlIdent: 'html.faq',
+    },
+  }),
+  SUPPORT: new PageParams({
+    name: 'support',
+
+    externalLink: null,
+
+    internalPage: {
+      pageRoute: '/support', // static, don't change '*/support'! internal page in case no external is defined
+      footerIdent: 'site.support', // localized string identifier
+      headTitleIdent: 'site.support', // localized string identifier
+      headlineIdent: 'site.support', // localized string identifier. on null it's hidden, on empty string default is used
+      hasContainer: true,
+      hasBaseCard: true,
+      hasLoginInHeader: true,
+      // in case internal page content is here 'webapp/locales/html/'
+      htmlIdent: 'html.support',
+    },
+  }),
+}
```

Comment:
Can this be a new file please?

Reply:
I put this classic in a separate file now.
Thanks for the hint, Ulf. 😍